react/README.md

```diff
@@ -701,6 +702,15 @@ We don’t recommend using indexes for keys if the order of items may change.
   > Why? [`isMounted` is an anti-pattern][anti-pattern], is not available when using ES6 classes, and is on its way to being officially deprecated.
 
   [anti-pattern]: https://facebook.github.io/react/blog/2015/12/16/ismounted-antipattern.html
+  
+## Things to avoid
```

Comment:
This isn't the right phrasing here; someone using React 16.2 shouldn't be avoiding anything.